packages/api/lib/rulesHelpers.js

```diff
@@ -104,11 +104,48 @@ function lookupCollectionInEvent(eventObject) {
   });
 }
 
+/**
+ * Build payload from rule for lambda invocation
+ *
+ * @param {RuleRecord} rule           - API rule
```

Comment:
Mission accomplished: https://github.com/nasa/cumulus/pull/3149/commits/f901007e0cf6d88ef39754f60b62ae4489d44a7e

We should discuss best practices here. It's probably worth adding an acceptance criteria on all these tickets to import/create the correct types if you're in a JS file. Otherwise it'll be a little scattered depending on who does the ticket or reviews if it actually gets done. I'll make a note for the next arch meeting.